doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -140,6 +140,17 @@ will route the requests to the followers, otherwise to the leader.
 |``
 |Yes^*^
 
+|`query.count`
+|Query count, used only in combination with `query` option, it's a query that returns a `count`
+field like the following
+```
+MATCH (p:Person)-[r:BOUGHT]->(pr:Product)
```

Comment:
see my optimized version in the other comment.

Reply:
also typo in RETURN

Reply:
Can I also opt out of the count query and just provide a literal value if I know how much data is there?
That would be really useful.

Reply:
Done